packages/block-library/src/pattern/block.json

```diff
@@ -6,13 +6,29 @@
 	"category": "theme",
 	"description": "Show a block pattern.",
 	"supports": {
-		"html": false,
-		"inserter": false
```

Comment:
Curious to know why we want to allow the pattern block to be shown in the inserter? Moreover, should it be allowed to create a pattern block without the `slug` attribute? What's the difference between it and a group block?